tests/views_integration/README.md

```diff
@@ -322,4 +322,92 @@ This second test example shows what is, probably, the main feature of data-drive
 @todo @paulmskim
 
 ### Other Testing
-@todo @bordoni
+
+Besides all the tests mentioned above due to the multiple moving parts of the software is important to make sure we test unwanted behavior to make sure they are not being "fixed" in a future bug report.
+
+The test below is dealing with the multiple possibilities of the template bootstraping, and how it is based on an admin setting, which can be changed in a lot of unpredictable ways, so we bake into our integration testing a way to make sure even if those bad values are pulled from the database we still continue to behave as expoected.
```

Comment:
```suggestion
In the test below we cover the class bootstrapping the templates with tests to ensure it handles all the possible admin setting combinations, good and bad alike, in a predictable way.
Any change of the code causing a test failure would signal a change in the class behavior we want to avoid.
```